Tests/Marketplace/marketplace_services.py

```diff
@@ -1616,6 +1617,10 @@ def collect_content_items(self):
                             'dataType': content_item.get('dataType', ""),
                             'widgetType': content_item.get('widgetType', "")
                         })
+                    elif current_directory == PackFolders.LISTS.value:
+                        folder_collected_items.append({
+                            'name': content_item.get('name', "")
+                        })
```

Comment:
what about description?

Reply:
Only name as can be seen here: https://github.com/demisto/server/blob/6a8074b45de1f8ccc53cd774dc02a9a1d6e95623/domain/contentpack.go#L343